client/driver/docker_test.go

```diff
@@ -2458,3 +2458,33 @@ func TestDockerDriver_AdvertiseIPv6Address(t *testing.T) {
 		t.Fatalf("Got GlobalIPv6address %s want GlobalIPv6address with prefix %s", expectedPrefix, container.NetworkSettings.GlobalIPv6Address)
 	}
 }
+
+func TestDockerDriver_authFromHelper(t *testing.T) {
+	dir, err := ioutil.TempDir("", "test-docker-driver_authfromhelper")
+	require.NoError(t, err)
+	defer os.RemoveAll(dir)
+	helperPayload := "{\"Username\":\"hashi\",\"Secret\":\"nomad\"}"
+	helperContent := []byte(fmt.Sprintf("#!/bin/sh\ncat > %s/helper-$1.out;echo '%s'", dir, helperPayload))
```

Comment:
Guard this test from running on windows?

Reply:
Do build tags work on tests? Or should I skip if `runtime.GOOS == "windows"`?

Reply:
Build tags work on test.